ruby/tests/test_ruby_package.proto

```diff
@@ -2,6 +2,6 @@ syntax = "proto3";
 
 package foo_bar;
 
-option ruby_package = "A.B";
+option ruby_package = "A::B";
```

Comment:
Does this change the existing API?

Reply:
Sort of, but I think the original implementation was incorrect. 

In all other languages, the package options follow the same format as the language and `::` is the correct delimiter in Ruby.

For example, this proto has a bunch of language package options defined: https://github.com/googleapis/googleapis/blob/1a4f0f12777dc2f8bf2c2ce84438329639c75e29/google/pubsub/v1/pubsub.proto#L33

Reply:
Will this break existing users?
If so, can we support both for now?

Reply:
Maybe in protoc, we can throw warning if "A.B" is used.

Reply:
Supporting both options sounds like a good solution. I've updated it to work with both and issue a warning if the `A.B` form is used instead of `A::B`.